### PR TITLE
Rebound rev3

### PR DESCRIPTION
**Pull request by Rossman360**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ x] New feature
- [ ] Enhancement/optimization
- [ x] Keyboard (addition or update)
- [ x] Keymap/layout/userspace (addition or update)
- [ x] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [ x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by Rossman360** [comment]
looking over all this, I'm not sure why it still has the ortho 4x12 layout so maybe I should just delete all that

File: keyboards/montsinger/rebound/rev3/keymaps/rossman360/keymap.c [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+[_NUM] = LAYOUT_all(
+    KC_GRV , KC_1   , KC_2   , KC_3   , KC_4   , KC_5   ,          KC_6   , KC_7   , KC_8   , KC_9   , KC_MINS, KC_EQL,
+    KC_CAPS, _______, _______, _______, _______, _______, _______, KC_MINS, KC_4   , KC_5   , KC_6   , KC_COLN, _______,
+    _______, _______, _______, _______, _______, KC_LBRC, _______, KC_RBRC, KC_1   , KC_2   , KC_3   , KC_BSLS, _______,
+    _______, _______, _______, _______, _______, _______, _______, KC_NO  , KC_NO  , KC_0   , _______, _______, _______
+  )
+};
+
+void encoder_update_user(uint8_t index, bool clockwise) {
+   switch(biton32(layer_state)){
```
Comment (fauxpark) [inline keyboards/montsinger/rebound/rev3/keymaps/rossman360/keymap.c:58]:
```suggestion
   switch(get_highest_layer(layer_state)){
```

File: keyboards/montsinger/rebound/readme.md [hunk 2]
```diff
@@ -1,16 +1,17 @@
 # Rebound
 
-![Rebound](https://imgur.com/zMPhNmm.jpg)
+![Rebound](https://i.imgur.com/IoV64hC.jpg)
```
Comment (noroadsleft) [inline keyboards/montsinger/rebound/readme.md:3]: Swaps the image for one with fewer pixels.

```suggestion
![Rebound](https://i.imgur.com/IoV64hCl.jpg)
```

File: keyboards/montsinger/rebound/rev3/info.json [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+                {"x":11, "y":0},
+                {"x":12, "y":0},
+
+                {"x":0, "y":1},
+                {"x":1, "y":1},
+                {"x":2, "y":1},
+                {"x":3, "y":1},
+                {"x":4, "y":1},
+                {"x":5, "y":1},
+                {"x":6, "y":2},
```
Comment (noroadsleft) [inline keyboards/montsinger/rebound/rev3/info.json:30]:
```suggestion
                {"x":6, "y":1},
```

File: keyboards/montsinger/rebound/rev3/keymaps/default/keymap.c [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+
+[_RAISE] = LAYOUT_all(
+  KC_GRV,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,             KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_DEL,
+  KC_DEL,  KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   _______, KC_F6,   KC_MINS, KC_EQL,  KC_LBRC, KC_RBRC, KC_BSLS,
+  _______, KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  _______, KC_F12,  KC_NUHS, KC_NUBS, _______, _______, _______,
+  _______, _______, _______, _______, _______, _______, _______, _______, _______, KC_MNXT, KC_VOLD, KC_VOLU, KC_MPLY
+),
+
+[_ADJUST] =  LAYOUT_all(
+  _______, RESET,   _______, _______, _______,          _______, _______, _______, _______, _______, _______, KC_DEL,
```
Comment (noroadsleft) [inline keyboards/montsinger/rebound/rev3/keymaps/default/keymap.c:65]:
```suggestion
  _______, RESET,   _______, _______, _______, _______,          _______, _______, _______, _______, _______, KC_DEL,
```

File: keyboards/montsinger/rebound/rev3/keymaps/default/keymap.c [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+  _______, _______, _______, AU_ON,   AU_OFF,  _______, AG_NORM, AG_SWAP, QWERTY,  COLEMAK, DVORAK,  _______, _______,
```
Comment (noroadsleft) [inline keyboards/montsinger/rebound/rev3/keymaps/default/keymap.c:66]: Right now you have `AG_NORM` assigned to the top key of the center column, and I'm not convinced that's what you had planned.

**Comment by noroadsleft** [comment]
Thanks!

**Pull request merged**